pkg/asset/agent/installconfig.go

```diff
@@ -309,6 +309,11 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 		logrus.Warnf(fmt.Sprintf("%s is ignored", fieldPath))
 	}
 
+	if installConfig.FeatureSet != configv1.Default {
```

Comment:
Isn't maybe a little bit too wide as a check? AFAICS the featureSet field is also potentially consumed by the [bootstrapTemplateData](https://github.com/openshift/installer/blob/271f2565c6449c557d50373469635d83eb8b5026/pkg/asset/ignition/bootstrap/common.go#L69C6-L69C27), which goes up to the various bootstrap asset used at the end by the `create ignition-configs` command exercised by the assisted service

Reply:
That's a different install-config though. That one always has the default because we don't pass this value through yet ([AGENT-554](https://issues.redhat.com/browse/AGENT-554)).